pkg/network/network.go

```diff
@@ -118,6 +118,10 @@ func Render(conf *cnao.NetworkAddonsConfigSpec, manifestDir string, openshiftNet
 	if err != nil {
 		return nil, err
 	}
+
+	for _, obj := range o {
+		log.Printf("kmp obj: %+v", *obj)
```

Comment:
This does not look like a proper log we'd like to have in our stderr. Either name it appropriately or delete it. Note that this may spam the log a lot.